pkg/controllers/customeradmin/group_reconcile.go

```diff
@@ -0,0 +1,66 @@
+package customeradmin
+
+import (
+	"strings"
+
+	"github.com/Azure/azure-sdk-for-go/services/graphrbac/1.6/graphrbac"
+	"github.com/pkg/errors"
+	"github.com/sirupsen/logrus"
+	meta_v1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+)
+
+var (
+	protectedUsers = map[string]struct{}{
+		"customer-cluster-admin":  {},
+		"customer-cluster-reader": {},
+	}
+)
+
+func reconcileOneGroup(log *logrus.Entry, gc *graphrbac.GroupsClient, cli *openshiftClient, group graphrbac.ADGroup, kgroupName string, msGroupMembers []graphrbac.AADObject) error {
+	kubeGroup, err := cli.UserV1.Groups().Get(kgroupName, meta_v1.GetOptions{})
+	if err != nil {
+		if strings.Contains(err.Error(), "not found") {
+			kubeGroup = nil
+		} else {
+			return errors.Wrapf(err, "Groups().Get(%s)", kgroupName)
```

Comment:
Wrapping api errors is ugly and breaks the api error type. Also, this new message does not look like it adds anything new so I'd remove.

Reply:
ok